src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -0,0 +1,109 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceContacts;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * ContactSteps.java
+ * Class for steps of account.feature.
+ */
+public class ContactSteps {
+    private WebDriver webDriver;
+    private SalesForceContacts salesForceContacts;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "salesForceContacts" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)